index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+   <div class="Container">
+     <div class="header">
```

Comment:
- [ ] There is no need for an extra `div`. kindly remove line 11.
- [ ] The previous reviewer ask to use semantic tag here, you misunderstood the requirement. KIndly use like this:-
`<header class="header">`